docs/gateway-configuration/vlan-configuration.md

```diff
@@ -0,0 +1,144 @@
+                <esf:value>false</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.type" type="String">
+                <esf:value>VLAN</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.ip4.gateway" type="String">
+                <esf:value>192.168.41.254</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.ip4.status" type="String">
+                <esf:value>netIPv4StatusEnabledWAN</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.vlanFull.config.ip4.dnsServers" type="String">
+                <esf:value>8.8.8.8</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.vlan.parent" type="String">
+                <esf:value>ens33</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.vlan.id" type="Integer">
+                <esf:value>41</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.ip4.address" type="String">
+                <esf:value>192.168.41.1</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.ip4.prefix" type="Short">
+                <esf:value>24</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.vlan.flags" type="Integer">
+                <esf:value>1</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.vlan.ingress" type="String">
+                <esf:value>1:2,3:4</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="net.interface.ens33.41.config.vlan.egress" type="String">
+                <esf:value>5:6</esf:value>
+            </esf:property>
+        </esf:properties>
+    </esf:configuration>
+</esf:configurations>
+```
+
+
+## VLAN Management
+Once a VLAN is created it can be managed via the Kura UI just like any other Ethernet interface.
+
+![VLAN UI management](./images/vlan-interface-example.png)
+
+!!! warning
+    Setting a VLAN status to Disabled deletes its configuration in NetworkManager and the related virtual interface from the system. Although it will is no longer be visible on the UI, all the configurations are left in Kura. Therefore the VLAN can be restored by setting the `net.interface.<interface>.config.ip4.status` to `netIPv4StatusEnabledLAN` or `netIPv4StatusEnabledWAN` via snapshot upload, then resume configuration via UI.
```

Comment:
```suggestion
    Setting a VLAN status to "_Disabled_" deletes its configuration in NetworkManager and the related virtual interface from the system. Although it will is no longer be visible on the UI, all the configurations are left in Kura. Therefore the VLAN can be restored by setting the `net.interface.<interface>.config.ip4.status` to `netIPv4StatusEnabledLAN` or `netIPv4StatusEnabledWAN` via snapshot upload, then resume configuration via UI.
```